src/app/dashboard/dashboard.service.mock.ts

```diff
@@ -75,15 +73,12 @@ export const campaigns: Campaign[] = [
     type: 'house',
     repName: 'Jingleheimer',
     notes: '',
-    flights,
     loading: false,
     actualCount: 125000,
     totalGoal: 100000
   }
 ];
 
-flights.forEach(flight => (flight.parent = campaigns[0]));
-
```

Comment:
Another change to try to address that tests are hanging on. Did not fix the problem.
```
(node:85206) UnhandledPromiseRejectionWarning: TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'Object'
    |     property 'element' -> object with constructor 'Object'
    |     property 'publicProviders' -> object with constructor 'Object'
    |     property 'ɵNgNoValidate_122' -> object with constructor 'Object'
    --- property 'parent' closes the circle
    at JSON.stringify (<anonymous>)
    at process.target._send (internal/child_process.js:778:25)
    at process.target.send (internal/child_process.js:676:19)
    at reportSuccess (/Users/sandikbarr/code/grove.prx.org/node_modules/jest-worker/build/workers/processChild.js:67:11)
(node:85206) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:85206) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
Cool, cool. This was not the problem. Still probably a valid change though in general, so keeping it.